aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -97,27 +97,27 @@ For more information on disabling modules with configuration settings, follow th
 
-2. Deploy the app without a **\<modules>** section in *web.config*. If an app is deployed with a *web.config* containing the **\<modules>** section without having unlocked the section first in the IIS Manager, the Configuration Manager throws an exception when attempting to unlock the section. Therefore, deploy the app without a **\<modules>** section.
+2. Deploy the app without a `<modules>` section in *web.config*. If an app is deployed with a *web.config* containing the `<modules>` section without having unlocked the section first in the IIS Manager, the Configuration Manager throws an exception when attempting to unlock the section. Therefore, deploy the app without a `<modules>` section.
 
-3. Unlock the **\<modules>** section of *web.config*. In the **Connections** sidebar, select the website in **Sites**. In the **Management** area, open the **Configuration Editor**. Use the navigation controls to select the `system.webServer/modules` section. In the **Actions** sidebar on the right, select to **Unlock** the section.
+3. Unlock the `<modules>` section of *web.config*. In the **Connections** sidebar, select the website in **Sites**. In the **Management** area, open the **Configuration Editor**. Use the navigation controls to select the `system.webServer/modules` section. In the **Actions** sidebar on the right, select to **Unlock** the section.
 
-4. At this point, a **\<modules>** section can be added to the *web.config* file with a **\<remove>** element to remove the module from the app. Multiple **\<remove>** elements can be added to remove multiple modules. If *web.config* changes are made on the server, immediately make the same changes to the project's *web.config* file locally. Removing a module this way won't affect the use of the module with other apps on the server.
+4. At this point, a `<modules>` section can be added to the *web.config* file with a `<remove>` element to remove the module from the app. Multiple `<remove>` elements can be added to remove multiple modules. If *web.config* changes are made on the server, immediately make the same changes to the project's *web.config* file locally. Removing a module this way won't affect the use of the module with other apps on the server.
 
    ```xml
-   <configuration> 
-    <system.webServer> 
-      <modules> 
-        <remove name="MODULE_NAME" /> 
-      </modules> 
-    </system.webServer> 
+   <configuration>
+    <system.webServer>
+      <modules>
+        <remove name="MODULE_NAME" />
+      </modules>
+    </system.webServer>
    </configuration>
    ```
 
-An IIS module can also be removed with *Appcmd.exe*. Provide the `MODULE_NAME` and `APPLICATION_NAME` in the command:
+An IIS module can also be removed with `Appcmd.exe`. Provide the `MODULE_NAME` and `APPLICATION_NAME` in the command:
```

Comment:
Use italics for file names.

Reply:
I guess I was thinking ... 'this is a command' ... but yes ... that was incorrect.